src/components/fpl/MatchResults.tsx

```
@@ -320,611 +337,510 @@ export default function MatchResults({
-                  />
-                  <div className="text-sm lg:text-base font-bold text-gray-900 dark:text-white">
-                    {match.home_team.short_name}
-                  </div>
-                </div>
-                <div className="text-sm lg:text-base text-gray-600 dark:text-gray-400">
-                  VS
-                </div>
-                <div className="flex items-center gap-2">
-                  <div className="text-sm lg:text-base font-bold text-gray-900 dark:text-white">
-                    {match.away_team.short_name}
+                  {/* Status and Expand Icon */}
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)